test/e2e/resize_nodes.go

```diff
@@ -223,6 +257,59 @@ func waitForPodsCreatedRunningResponding(c *client.Client, ns, name string, repl
 	return nil
 }
 
+// Blocks outgoing network traffic on 'node'. Then verifies that 'podNameToDisappear',
+// that belongs to replication controller 'rcName', really disappeared.
+// Finally, it checks that the replication controller recreates the
+// pods on another node and that now the number of replicas is equal 'replicas'.
+// At the end (even in case of errors), the network traffic is brought back to normal.
+// This function executes commands on a node so it will work only for some
+// environments.
+func performTemporaryNetworkFailure(c *client.Client, ns, rcName string, replicas int, podNameToDisappear string, node *api.Node) {
+	Logf("Getting external IP address for %s", node.Name)
+	host := ""
+	for _, a := range node.Status.Addresses {
+		if a.Type == api.NodeExternalIP {
+			host = a.Address + ":22"
+			break
+		}
+	}
+	By(fmt.Sprintf("block network traffic from node %s to the master", node.Name))
+
+	defer func() {
+		By(fmt.Sprintf("Unblock network traffic from node %s to the master", node.Name))
+		// delete OUTPUT rule number 1
+		undropCmd := "sudo iptables --delete OUTPUT 1"
```

Comment:
Ok for most of the comments.
But I don't know how to find the IP of the master within the cluster. I'll leave the DNS name for now.